packages/shared/src/components/ResponsiveImage/ResponsiveImage.tsx

```diff
@@ -39,13 +44,13 @@ export const ResponsiveImage = ({
     return (
         <img
             data-test-id="image-block-img"
-            className="tw-flex tw-w-full"
+            className={joinClassNames(['tw-flex tw-w-full', className])}
             loading="lazy"
             src={sourceOptimised}
             width={imageWidth}
             height={imageHeight}
-            style={{ maxWidth: imageWidth }}
-            alt={altText}
+            style={style}
+            alt={altText ?? ''}
```

Comment:
Can be done in a different pr but I think it would actually be better to enforce altText as a required string type. A user could add an empty string but this would force people to at least think about it, as it's very commonly a forgotten attribute

Reply:
I included it in this pr 👍